### PR TITLE
Review lodestar cli flags

### PR DESCRIPTION
**Pull request by nflaig**
I did a quick review of Lodestar CLI flags

- removed --force as it should no longer be required
- updated deprecated flags

### CODE REVIEW

File: lodestar-vc-only.yml [hunk 1]
```diff
@@ -66,7 +66,6 @@ services:
       - ${LOG_LEVEL}
       - --suggestedFeeRecipient
       - ${FEE_RECIPIENT}
-      - --force
```
Comment (nflaig) [inline lodestar-vc-only.yml:69]: I haven't checked but I'd assume this flag was added to allow VC to start up after an unexpected shutdown (e.g. due to power loss)

With the next release this won't be required as we improved lockfile handling
- https://github.com/ChainSafe/lodestar/pull/6363

Better to remove this flag

File: lodestar.yml [hunk 2]
```diff
@@ -164,13 +163,12 @@ services:
...
       - /var/lib/lodestar/validators
-      - --server
+      - --beaconNodes
       - http://consensus:5052
-      -  --logLevel
+      - --logLevel
       - ${LOG_LEVEL}
       - --network
       - ${NETWORK}
-      - --force
```
Comment (nflaig) [inline lodestar.yml:173]: For voluntary exit this is also not required any more, the command can be executed while the VC is running
- https://github.com/ChainSafe/lodestar/pull/5950

File: lodestar-vc-only.yml (hunk 1 above)
Comment (yorickdowne) [reply lodestar-vc-only.yml:69]: Yes, it was added to deal with unclean shutdown like a power loss.

**Comment by yorickdowne** [comment]
This will require which Lodestar version?

**Comment by nflaig** [comment]
> This will require which Lodestar version?

It will work with any of the more recent releases but to get the stale lockfile fix in, would have to wait for 1.15.1, we have just cut a rc for this, will be released this week.

Will mark this as ready once the release is out

**Comment by yorickdowne** [comment]
OK. I may want to bundle this with #1682 (Besu 14.1.2) so that I don't have to increment the Eth Docker major twice, as both are breaking changes.

**Comment by nflaig** [comment]
[v1.15.1](https://github.com/ChainSafe/lodestar/releases/tag/v1.15.1) has been released

**Comment by yorickdowne** [comment]
Great! I’d like to hold on to this for a bit, see what ACDC says today about client releases. If I can bundle this breaking change with other breaking changes, that’d be nice. Saves me from needing to rapidly increase the version with every change.

**Pull request merged**